src/Data/Units/Time.purs

```diff
@@ -22,10 +22,14 @@ day = makeNonStandard "day" "d" (24.0 * 3600.0) second
 week ∷ DerivedUnit
 week = makeNonStandard "week" "week" (7.0 * 24.0 * 3600.0) second
 
--- | Unit of time, *1 month = 30 days + 10 hours + 30 minutes*.
+-- | Unit of time, *1 month = 30 days + 10 hours + 29 minutes + 10 seconds*.
 month ∷ DerivedUnit
-month = makeNonStandard "month" "month" ((30.0 * 24.0 + 10.5) * 3600.0) second
+month = makeNonStandard "month" "month" ((30.0 * 24.0 + 10.485) * 3600.0) second
 
--- | Unit of time, *1 year = 365.25 days* (Julian year).
+-- | Unit of time, *1 year = 365.2425 days* (Gregorian year).
 year ∷ DerivedUnit
-year = makeNonStandard "year" "year" (365.25 * 24.0 * 3600.0) second
+year = makeNonStandard "year" "year" (365.2425 * 24.0 * 3600.0) second
+
+-- | Unit of time, *1 julianyear = 365.25 days*.
+julianyear ∷ DerivedUnit
+julianyear = makeNonStandard "julianyear" "julianyear" (365.25 * 24.0 * 3600.0) second
```

Comment:
Should the name of the unit be `juilanyear` or `julianYear`? I could find no consistent choice regarding units with compound names; light year has a name of `lightyear`, while nautical mile is `nauticalMile`.

Also, should the long name of the unit be `julianyear` or julian year`? `nauticalMile` has `nautical mile` as a long name, while `lightyear` has no space in its long name.

Reply:
> Should the name of the unit be `juilanyear` or `julianYear`? I could find no consistent choice regarding units with compound names; light year has a name of `lightyear`, while nautical mile is `nauticalMile`.

Hm. "lightyear" can really be spelled that way (https://en.wikipedia.org/wiki/Light-year). For everything else, I would go with the camelCase variant. So I would propose we use "julianYear".

> Also, should the long name of the unit be `julianyear` or julian year? `nauticalMile` has `nautical mile` as a long name, while `lightyear` has no space in its long name.

Let's use the space for the camelCase variants. So "julian year".